### DISCUSSION TITLE
Can a human level intelligence be really controlled by human? Will artificial intelligence develop to a even higher level? Or are humans at the top of intelligence?

### DISCUSSION THREAD

**Discussion by lets-make-safe-ai** [lets-make-safe-ai/make-safe-ai#17] [General]
Can a human level intelligence be really controlled by human? Will artificial intelligence develop to a even higher level? Or are humans at the top of intelligence?

**Comment by chirag127** [comment]
I human will be always be at the top atleast until my death.
